rust/rope/src/find.rs

```diff
@@ -76,7 +76,7 @@ pub fn find(cursor: &mut Cursor<RopeInfo>, cm: CaseMatching, pat: &str) -> Optio
 /// 
 /// [find]: fn.find.html
 pub fn find_progress(cursor: &mut Cursor<RopeInfo>, cm: CaseMatching, pat: &str,
-    num_steps: usize) -> FindResult
+                     num_steps: usize) -> FindResult
```

Comment:
These are just whitespace changes? Maybe we should institute rustfmt so we don't have to have these discussions in PR's. It looks like this is a change from "block" to "visual" for fn args, in the [rustfmt lingo](https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#function-arguments). The former is the default, and also my preference, as it allows more room for arguments.

Reply:
I got confused here. I saw that in many files the "visual" style is used. Now that I look at some other files, I also see the "block" style is used. I could certainly change it back if that is wished.

Reply:
Yea, I tend to use visual indenting, which I prefer for oblique aesthetic reasons, although as a general rule I  try not to change whitespace in a file unless it's just clearly _wrong_.

 I have a bunch of similar rust style idiosyncrasies that I would be happy to abandon if we want to just go ahead and punt to rustfmt.

Reply:
Okay, should I change it back for this to get merged?

Reply:
I don't have a strong feeling and am ok with it being merged either way.